docs/Group GitHub Repos/Repos associated with Publications.md

```diff
@@ -0,0 +1,38 @@
+# Repos Associated with Publications
+
+## Description
+Below you will find the links to repos that are associated with different publications.
+
+## List of Repos 
```

Comment:
This heading is a bit confusing because you also have a list of repos below.  Insted I would use the headings I sent you in slack 'Toolkits and Packages' and 'Repos Associated with Publications'

Also for each of these you should change your hyperlinks to match what I had in slack. i.e. one hyperlink for the repo and one for the publicaitons.  Note that you can just put [] around the description and yperlink it to the github repo that way you don't need to type out the repo name itself.